{"uuid": "bfa234e3-216f-49cf-912e-5ba43b3a964d", "children": ["efa2ef83-2ad9-43db-a5a7-4416164f128f"], "befores": [{"name": "authorization", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"button_button__33qZ0 button_button_type_primary__1O7Bx button_button_size_medium__3zxIa\"> is not clickable at point (634,549) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/tests/conftest.py\", line 39, in authorization\n    auth_page.click_enter_button()\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/pages/auth_page.py\", line 24, in click_enter_button\n    self.find_element(self.enter_button_locator, 15).click()\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Перейти на по урлу", "status": "passed", "start": 1727547806799, "stop": 1727547807662}, {"name": "Заполнить форму авторизации", "status": "passed", "steps": [{"name": "Найти элемент", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \"//input[@name='name']\")"}, {"name": "time", "value": "10"}], "start": 1727547807662, "stop": 1727547807677}, {"name": "Найти элемент", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \"//input[@type='password']\")"}, {"name": "time", "value": "10"}], "start": 1727547807716, "stop": 1727547807721}], "parameters": [{"name": "email", "value": "'mir_48@example.org'"}, {"name": "password", "value": "'MrPBGmEU&0'"}], "start": 1727547807662, "stop": 1727547807727}, {"name": "Нажать на кнопку входа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class=\"button_button__33qZ0 button_button_type_primary__1O7Bx button_button_size_medium__3zxIa\"> is not clickable at point (634,549) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/pages/auth_page.py\", line 24, in click_enter_button\n    self.find_element(self.enter_button_locator, 15).click()\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/mchvell/Diplom_3/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Найти элемент", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \"//button[text()='Войти']\")"}, {"name": "time", "value": "15"}], "start": 1727547807727, "stop": 1727547807732}], "start": 1727547807727, "stop": 1727547807733}], "start": 1727547806798, "stop": 1727547807734}], "start": 1727547806798, "stop": 1727547807749}